PennyMe/Assets.xcassets/freigestellt_FMO.imageset/Contents.json

```diff
@@ -15,7 +16,7 @@
     }
   ],
   "info" : {
-    "version" : 1,
-    "author" : "xcode"
+    "author" : "xcode",
+    "version" : 1
   }
```

Comment:
Maybe this change and the updated image below caused the starting screen to break?

Reply:
Looks like only the two lines are switched, would be weird if that's the reason. No idea why the image is changed, will check later